src/Config/ObjectModel/RelationshipDefinitionSource.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+/// <summary>
+/// Identifies the source of the relationship.
+/// </summary>
+public enum RelationshipDefinitionSource
```

Comment:
I see multiple references to this where you're setting the `ForeignKeyDefinition.Source` in the logic, those references are never used.

Reply:
good catch. I wanted to use `RelationshipDefinitionSource` to supplement what we know about a particular FKDefinition. ultimately, didn't use it. Removing. But may be good to add in future.